packages/matchbox/src/components/QueryCard/QueryCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
+import Title from './Title';
+import Operator from './Operator';
+import Legend from './Legend';
+import Subtitle from './Subtitle';
+import Group from './Group';
+import { getChild } from '../../helpers/children';
+import { Close } from '@sparkpost/matchbox-icons';
+
+export type QueryCardProps = {
+  children?: React.ReactNode;
+  id?: string;
+  onClose?: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+const QueryCard = React.forwardRef<HTMLDivElement, QueryCardProps>(function QueryCard(
+  props,
+  userRef,
+) {
+  const { children, id, onClose } = props;
+
+  const hasLegend = Boolean(getChild('QueryCard.Legend', children).length);
+
+  return (
+    <Box id={id} ref={userRef} borderRadius="100" border="400" p="200" position="relative">
+      <Box display="flex" alignItems="flex-start">
+        {hasLegend && (
+          <Box flex="0" pr="200">
+            {getChild('QueryCard.Legend', children)}
+          </Box>
+        )}
+        <Box flex="1" mt="-2px">
+          {getChild('QueryCard.Title', children)} {getChild('QueryCard.Operator', children)}
+          <Box>{getChild('QueryCard.Subtitle', children)}</Box>
+        </Box>
+      </Box>
+      <Box>
+        {onClose && (
+          <Box position="absolute" top="100" right="100">
+            <Button size="small" variant="text" onClick={onClose} px="200">
```

Comment:
Didn't match the proposal mocks here, but thought it'd be easier to reuse Button instead of building a custom close button